sapi/cli/tests/gh11146.phpt

```diff
@@ -0,0 +1,45 @@
+--TEST--
+GH-11146: The built-in webserver will add ";charset=UTF-8" to the end of the Content-Type response header of the SSE response.
+--SKIPIF--
+<?php
+include "skipif.inc";
+?>
+--FILE--
+<?php
+include "php_cli_server.inc";
+php_cli_server_start(
+    <<<'CODE'
+header('Content-Type: text/event-stream');
+for ($i = 0; $i < 3; ++$i) {
+    echo 'data: ' . $i . "\n\n";
+}
+CODE
+);
+
+$host = PHP_CLI_SERVER_HOSTNAME;
+
+$fp = php_cli_server_connect();
+if(fwrite($fp, <<<HEADER
```

Comment:
would be nice if you could define heredoc outside `if` to be more readable. Also there should be a space after `if`